patterns/1-initial/change-the-middle-management-mindset.md

```diff
@@ -4,90 +4,126 @@ Change the Middle-Management Mindset
 
 ## Patlet
 
-TBD
+Middle managers often resist InnerSource due to misunderstanding and misaligned incentives. To overcome this resistance, educate them on the benefits, include InnerSource participation in their performance metrics, and demonstrate how it enhances their reputation, reduces team friction, and boosts overall productivity.
 
 ## Problem
 
-The InnerSource program does not live up to its expectations because middle management is reluctant to allocate resources to it. Expectations of the program are faster go-to-market, increased quality, reduced duplicative development, better integration capabilities, and increased developer satisfaction.
+The InnerSource program does not live up to its expectations because middle management is reluctant to allocate resources to it. Expectations of the program are faster go-to-market, increased quality, reduced duplicative development, better integration capabilities, and increased developer satisfaction. Middle management often misunderstands InnerSource's value, fears losing control, and struggles to understand how it fits their metrics. This gap prevents developers from contributing, hindered by managers' misaligned incentives and lack of strategic understanding.
+
+## Story
+
+A software development team at a large enterprise was eager to join an InnerSource project benefiting multiple teams. However, their middle manager hesitated to allocate time, citing quarterly deadlines and unclear ROI. Despite the developers' enthusiasm and potential for code reuse, theusing frustration and missed collaboration opportunities.
```

Comment:
Typo in "theusing". What did you mean to say here?

Reply:
Yes, fixed.